### PR TITLE
improvement - add skeleton screen and JavaScript disabled interstitial for Editor

### PR DESCRIPTION
**Pull request by elisechant**

### CODE REVIEW

**Comment by elisechant** [comment]
screen shot 2016-10-18 at 8 47 16 pm

skeleton2

**Comment by elisechant** [comment]
if JavaScript is disabled, show this message instead of rendering the app: 

screen shot 2016-10-19 at 7 13 27 am

File: config/initializers/assets.rb [hunk 1]
```diff
@@ -9,6 +9,6 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 
-assets = %w(dashboard.js login.js editor.js dashboard.css login.css editor.css gov-performance.svg coatofarms.svg)
+assets = %w(dashboard.js login.js editor.js dashboard.css login.css editor.css gov-performance.svg coatofarms.svg editor-skeleton.png)
```
Comment (elisechant) [inline config/initializers/assets.rb]: @tobyhede i'm thinking this is not right 😕

**Comment by catherine-jones** [comment]
@elisechant is this ready for review? or is it still in progress?

**Comment by elisechant** [comment]
@catherine-jones ready for review.

File: config/initializers/assets.rb (hunk 1 above)
Comment (tobyhede) [inline config/initializers/assets.rb]: What do you mean?

Comment (tobyhede) [inline config/initializers/assets.rb]: Like, having to add all the time?
We could push these to app/assets and avoid having to pull in manually

Comment (elisechant) [inline config/initializers/assets.rb]: the image in the list - the last item

**Pull request merged**